.expeditor/config.yml

```diff
@@ -0,0 +1,21 @@
+# Documentation available at https://expeditor-docs.es.chef.io/
+
+# The name of the product keys for this product (from mixlib-install)
+product_key:
+  - harmony
+  - chef-server
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: eng-services-notify
+
+github:
+  # The Github Team primarily responsible for handling incoming Pull Requests.
+  maintainer_group: chef/engineering-services
+  # Which Github branches to build Omnibus releases from, and what versions
+  release_branch:
+    - master
+
+# These actions are taken, in order they are specified, anytime a Pull Request is merged.
+merge_actions:
```

Comment:
I'm not sure we want any merge_actions in this project.  Other projects will subscribe to this one and any necessary actions where happen there.